primitives/src/tendermint.rs

```diff
@@ -36,7 +41,7 @@ pub struct TendermintIdentifier {
 
 impl TendermintIdentifier {
     /// Maximum size in bytes for a `TendermintIdentifier` in binary serialization.
-    pub const MAX_SIZE: usize = 2 * nimiq_serde::U32_MAX_SIZE + TendermintStep::SIZE;
+    pub const MAX_SIZE: usize = nimiq_serde::U8_SIZE + 2 * 4 + TendermintStep::SIZE;
```

Comment:
Why not `U32_SIZE`?

Reply:
Because we explicitly serialize it as four bytes and not as the postcard serialization of a `u32`.